src/VirtualClient/VirtualClient.Main/profiles/PERF-APACHEBENCH.json

```diff
@@ -0,0 +1,50 @@
+{
+  "Description": "Apache HTTP Server Benchmarking Workload",
+  "MinimumExecutionInterval": "00:01:00",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "01:00:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64,win-x64,win-arm64",
+    "SupportedOperatingSystems": "Debian,Ubuntu,Windows"
+  },
+  "Actions": [
+    {
+      "Type": "ApacheBenchExecutor",
+      "Parameters": {
+        "Scenario": "Scenario1",
```

Comment:
Could you name Scenario1 to make it clear? May be use something like ApacheBench_N50000_C10 which represents the scenario better.